src/main/java/com/powsybl/network/map/NetworkMapController.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.map;
+
+import com.powsybl.network.map.model.LineMapData;
+import com.powsybl.network.map.model.SubstationMapData;
+import io.swagger.annotations.*;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + NetworkMapController.API_VERSION + "/")
+@Api(tags = "network-map-server")
+@ComponentScan(basePackageClasses = NetworkMapService.class)
+public class NetworkMapController {
+
+    public static final String API_VERSION = "v1";
+
+    @Inject
+    private NetworkMapService networkMapService;
```

Comment:
Why do you use inject annotation and autowired. It doesn't change anything but it's probably best to only use one of them and not mix them up.

Reply:
Agreed, I replaced with @Autowired